clients/client/README.md

```diff
@@ -90,7 +90,7 @@ let pc = await pulse.consume({
 
 The set of API entries listed below is generated from the built-in references.
 Detailed documentation with description, payload and result format details is
-available in the [docs reference section](https://docs.taskcluster.net/docs/reference).
+available in the [docs reference section](https://github.com/taskcluster/taskcluster/tree/master/libraries/references).
```

Comment:
The taskcluster-lib-references library and the "reference section" are different.  I think we can just drop the link here, leaving something like "..available in the reference section of the Taskcluster documentation"